src/jdk.jartool/share/man/jarsigner.md

```diff
@@ -925,6 +925,12 @@ hasExpiringTsaCert
 hasNonexistentEntries
 :   This JAR contains signed entries for files that do not exist.
 
+internalConsistenciesDetected
+
```

Comment:
I don't think you need a line break here, the other warnings don't.

Reply:
Removed line break.